pyproject.toml

```diff
@@ -125,6 +125,7 @@ disable = [
     "too-few-public-methods",
     "too-many-arguments",
     "too-many-locals",
+    "too-many-positional-arguments",
```

Comment:
So this is needed as pylist was complaining about too many PAs ?

Reply:
This seems to be a new flag (or maybe they reduced the number of this check) in a recent pylint version. I do not see a harm on deactivating it as some times we definitely need several positional arguments. Overall, I believe this should be a reviewer's task to assess when something is adequate or not.